### PR TITLE
ASU workshop save-the-date post

### PR DESCRIPTION
**Pull request by fiona-naughton**

### CODE REVIEW

**Comment by fiona-naughton** [comment]
@orbeckst, if you still have time for a look (also pinging @IAlibay @jennaswa ) - thanks!

File: _posts/2024-04-24-ASUworkshop.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+style="float: left; height: 20em; " />
+
+<img
+src="/public/images/MolSSI_Logo.png"
+title="MolSSI logo" alt="MolSSI logo"
+style="float: left; height: 5em; " />
+
+MDAnalysis, in conjunction with [MolSSI](https://molssi.org/), is happy to announce an 
+upcoming in-person workshop, to be held **June 25th-26th** at **Arizona State University** 
+(Tempe Campus). This two-day workshop is intended to build attendees from user to developer
```
Comment (orbeckst) [inline _posts/2024-04-24-ASUworkshop.md:18]: build -> build up ?

File: _posts/2024-04-24-ASUworkshop.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+<img
+src="/public/images/MolSSI_Logo.png"
+title="MolSSI logo" alt="MolSSI logo"
+style="float: left; height: 5em; " />
+
+MDAnalysis, in conjunction with [MolSSI](https://molssi.org/), is happy to announce an 
+upcoming in-person workshop, to be held **June 25th-26th** at **Arizona State University** 
+(Tempe Campus). This two-day workshop is intended to build attendees from user to developer 
+and will cover an introduction to MDAnalysis, software best practices, and guide participants 
+through the creation of their own [MDAKit]( https://www.mdanalysis.org/pages/mdakits).
```
Comment (orbeckst) [inline _posts/2024-04-24-ASUworkshop.md:20]: space ... may still work
```suggestion
through the creation of their own [MDAKit](https://www.mdanalysis.org/pages/mdakits).
```

**Review by orbeckst — APPROVED** [review]
Thanks! Love the saguaro! 🌵 

Minor comments and lgtm!

File: _posts/2024-04-24-ASUworkshop.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Save the Date - in-person workshop at Arizona State University
```
Comment (IAlibay) [inline _posts/2024-04-24-ASUworkshop.md:3]: I don't know how it renders, but a potential enhancement might be to add the date in the title for quick reading

**Review by IAlibay — APPROVED** [review]
lgtm!

File: _posts/2024-04-24-ASUworkshop.md (hunk 1 above)
Comment (RMeli) [reply _posts/2024-04-24-ASUworkshop.md:18]: Other possibilities: "transition", "transform"?

**Review by RMeli — APPROVED** [review]

File: _posts/2024-04-24-ASUworkshop.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+<img
+src="/public/images/ASUworkshop_cactus.png"
+title="MDAnalysis ASU workshop 2024" alt="MDAnalysis Workshop Just 25-26th Arizona State University"
```
Comment (tylerjereddy) [inline _posts/2024-04-24-ASUworkshop.md:8]:
```suggestion
title="MDAnalysis ASU workshop 2024" alt="MDAnalysis Workshop June 25-26th Arizona State University"
```

Might be a typo, noticed while forwarding to one of my postdocs

**Review by orbeckst — CHANGES_REQUESTED** [review]
Dates are 24/25 June! 

Blocking until I can fix these...

**Comment by orbeckst** [comment]
I am really sorry that I didn't catch the dates in my first review. I did a quick edit.

File: _posts/2024-04-24-ASUworkshop.md (hunk 4 above)
Comment (orbeckst) [reply _posts/2024-04-24-ASUworkshop.md]: thanks, fixed it

**Review by orbeckst — APPROVED** [review]

**Comment by orbeckst** [comment]
@fiona-naughton please squash-merge whenever you're happy. Thanks!

**Pull request merged**